src/Components/article-comments.jsx

```diff
@@ -0,0 +1,62 @@
+import { useContext, useEffect, useState } from "react";
+import UserContext from "../Contexts/user-context";
+import { getCommentsByArticleId, getUsers } from "../../api";
+
+
+function ArticleComments(props){
+
+    const { currentUser } = useContext(UserContext)
+    const [comments, setComments] = useState([])
+    const articleId = props.articleId
+    
+
+    useEffect(()=>{
+        getCommentsByArticleId(articleId)
+        .then((response)=>{
+            setComments(response.comments)
+        })
+    }, [currentUser])
+
```

Comment:
There doesn't seem to be any loading state indication on the articles page for the comments or the article itself? (Tested by setting a slow internet speed on the dev tools)
This is worth making some consideration for, especially because render backends will sleep when inactive and take a short while to spin up (In which time you'd want some user feedback)